project/src/main/webapp/index.html

```diff
@@ -76,17 +77,24 @@ <h1> FoodMood - your food order helper!</h1>
         <input type="radio" id="price-4" name="price" value="4">
         <label for="price-4">Very Expensive</label>
       </form>
+      <form id="old-new-form" class="form-element-toolbar" style="display: none">
+        <p>Mainly new recommendations?
```

Comment:
Phrasing suggestion - maybe use the world "explore"? 
Something like - 
Explore new places?

Reply:
great way to frame it! done :)